Gruntfile.js

```diff
@@ -43,16 +43,26 @@ module.exports = function (grunt) {
           'src/',
         ],
       },
-      'gts-check': {
+      lint: {
         cmd: 'node',
-        args: ['node_modules/gts/build/src/cli', 'check'],
+        args: ['node_modules/eslint/bin/eslint', 'src/**/*.ts'],
```

Comment:
Are all of the gts checks now encompassed by eslint? (not sure what the others were, maybe just prettier)

Reply:
Yea, eslint runs the gts ones and it extends

"eslint:recommended",
    "plugin:node/recommended",
    "prettier"